src/util/achievement-data.tsx

```diff
@@ -1,15 +1,15 @@
 export type Achievement = {
-  id: number;
+  id: string;
   title: string;
   description: string;
   icon?: string | undefined;
   progress: boolean;
 };
 
 export enum AchievementId {
-  DATENSCHUTZ_HELD,
-  WERBE_GURU,
-  PHISHING_FAENGER,
+  DATENSCHUTZ_HELD = "DATENSCHUTZ_HELD",
```

Comment:
why did you convert the id to a string?

Reply:
I need the string in the Database and not a number. it is not intuitive